packages/frontend/src/components/Buttons/CloseButton/CloseButton.tsx

```diff
@@ -4,6 +4,5 @@ interface CloseButtonProps {
     closeModal: () => void
 }
 
-export const CloseButton: React.FC<CloseButtonProps> = ({ closeModal }) => {
-    return <button className="small-button close-button center-child" onClick={closeModal}></button>
-}
+// eslint-disable-next-line jsx-a11y/control-has-associated-label, react/button-has-type
```

Comment:
I see that the react/button-has-type is being disabled a lot, why not disable it completely instead of for each line?

Reply:
because we should probably fix this for accessibility reasons